python/guide/getting-started/to-mxnet/pytorch.md

```diff
@@ -205,27 +191,24 @@ for epoch in range(5):
         epoch, total_loss/len(mx_train_data), time.time()-tic))
 ```
 
-Some of the differences in MXNet when compared to PyTorch are as follows:
+Some of the differences in Apache MXNet when compared to PyTorch are as follows:
 
-* In MXNet, you don't need to flatten the 4-D input into 2-D when feeding the
-data into forward pass.
+* In Apache MXNet, you don't need to flatten the 4-D input into 2-D when feeding the data into forward pass.
 
-* In MXNet, you need to perform the calculation within the `autograd.record()`
-scope so that it can be automatically differentiated in the backward pass.
+* In Apache MXNet, you need to perform the calculation within the [autograd.record()](https://beta.mxnet.io/api/gluon-related/_autogen/mxnet.autograd.record.html) scope so that it can be automatically differentiated in the backward pass.
 
-* It is not necessary to clear the gradient every time as with Pytorch's
-`trainer.zero_grad()` because by default the new gradient is written in, not
-accumulated.
+* It is not necessary to clear the gradient every time as with PyTorch's `trainer.zero_grad()` because by default the new gradient is written in, not accumulated.
 
-* You need to specify the update step size (usually batch size) when performing
-`step()` on the trainer.
+* You need to specify the update step size (usually batch size) when performing [step()](https://beta.mxnet.io/api/gluon/_autogen/mxnet.gluon.Trainer.step.html) on the trainer.
 
-* You need to call `.asscalar()` to turn a multidimensional array into a scalar.
+* You need to call [`.asscalar()`](https://beta.mxnet.io/api/ndarray/_autogen/mxnet.ndarray.NDArray.asscalar.html) to turn a multidimensional array into a scalar.
 
-* In this sample, MXNet is twice as fast as PyTorch. Though you need to be
-cautious with such toy comparisons.
+* In this sample, Apache MXNet is twice as fast as PyTorch. Though you need to be cautious with such toy comparisons.
 
 ## Conclusion
 
-MXNet and PyTorch share a lot of similarities. It makes PyTorch users much easy
-to use MXNet.
+As we saw above, Apache MXNet Gluon API and PyTorch are similar in use.
+
+## Recommended Next Steps
+
+While Apache MXNet Gluon API is very similar to PyTorch, there are some extra functionality using which can make your code even faster. Check out [Hybridize tutorial](https://beta.mxnet.io/guide/packages/gluon/hybridize.html) to learn how to write imperative code which can be converted to symbolic one. Also, check out how to extend Apache MXNet with your own [custom layers](https://beta.mxnet.io/guide/extend/custom_layer.html).
```

Comment:
Suggest having this as bullet points eg.
* check out [Hybridize tutorial] ....
* Also check out [custom layers] ...